### DISCUSSION TITLE
Toast.Make shows .Net image

### DISCUSSION THREAD

**Discussion by roysurles** [CommunityToolkit/Maui#1200] [Q&A] [answered]
How to display a toast without the .Net purple image?

**Comment by VladislavAntonyuk** [comment]
Toast only displays text. Could you send a screenshot?

**Comment by roysurles** [comment]
<PackageReference Include="CommunityToolkit.Maui" Version="5.1.0" />
<PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.0" />

var toast = Toast.Make("Sample Toast", CommunityToolkit.Maui.Core.ToastDuration.Long);
await toast.Show();


![image](https://github.com/CommunityToolkit/Maui/assets/37274814/bc1d501f-0f67-4113-8fd0-61a4729dd43c)

**Comment by VladislavAntonyuk** [comment]
I assume it’s a standard toast on Android 33. It displays the icon of your application so user understands which app sent toast.

**Comment by roysurles** [comment]
Ok, fair enough.  Is it possible to toast without the application icon or specify the icon?

**Reply by TheCodeTraveler** [reply, marked as answer]
Hey @roysurles! I understand what you're asking, but, no, this is just how Toast works. 

Here's the docs for Android showing the Toast API and its functionality: https://developer.android.com/guide/topics/ui/notifiers/toasts

Toast is implemented by Android at the Operating System level. Because a Toast can be displayed if the app is not in the foreground, Android includes its app icon to let the user know which app displayed the Toast. `CommunityToolkit.Maui` is just calling Android's Toast API.

**Reply by VladislavAntonyuk** [reply]
https://stackoverflow.com/questions/74784626/how-to-change-toast-messages-default-icon-in-jetpack-compose

**Comment by CliffAgius** [comment]
Closed as answered
